pulp_ansible/app/serializers.py

```diff
@@ -95,6 +95,21 @@ class CollectionRemoteSerializer(RemoteSerializer):
         required=False,
         allow_null=True,
     )
+    auth_url = serializers.CharField(
+        help_text=_(
+            "The URL of a Keycloak server ‘token_endpoint’ "
+            "if using SSO auth (Automation Hub for ex)."
+        ),
+        allow_null=True,
+        required=False,
+        max_length=255,
+    )
+    token = serializers.CharField(
+        help_text=_("The token key to use for authentication."),
```

Comment:
Can we also add a sentence like: `See https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client for more details`.